.github/workflows/java-cdk-tests.yml

```diff
@@ -69,3 +69,17 @@ jobs:
           concurrent: true
           # TODO: be able to remove the skipSlowTests property
           arguments: --scan :airbyte-cdk:check -DskipSlowTests=true
+
+  run-connector-tests:
+    needs:
+      - changes
+    if: needs.changes.outputs.java == 'true'
+    name: Run Java Connector Tests with Local CDK
+    uses: ./.github/workflows/connector-ci-checks.yml
+    with:
+      local-cdk-changed: true
```

Comment:
I think it might be a good idea to explicitly pass `gitref` and `repo` inputs here. I checked the workflow, and in theory it _should_ get the correct refs from context. I suggest it just because these things can sometimes be hard to debug if they end up pulling the wrong ref from context.